sale_by_packaging/models/sale_order_line.py

```diff
@@ -96,42 +78,26 @@ def _force_qty_with_package(self):
     def _onchange_product_uom_qty(self):
         self._force_qty_with_package()
         res = super()._onchange_product_uom_qty()
-        is_pack_multiple_warning = self._check_qty_is_pack_multiple()
-        if is_pack_multiple_warning:
-            self.product_packaging_qty = False
-            self.product_packaging = False
-        return res if res else is_pack_multiple_warning
-
-    def _check_qty_is_pack_multiple(self):
-        """ Check only for product with sell_only_by_packaging
-        """
-        # and we dont want to have this warning when we had the product
-        if self.product_id.sell_only_by_packaging:
-            if not self._get_product_packaging_having_multiple_qty(
-                self.product_id, self.product_uom_qty, self.product_uom
-            ):
-                warning_msg = {
-                    "title": _("Product quantity cannot be packed"),
-                    "message": _(
-                        "For the product {prod}\n"
-                        "The {qty} is not the multiple of any pack.\n"
-                        "Please add a package"
-                    ).format(prod=self.product_id.name, qty=self.product_uom_qty),
-                }
-                return {"warning": warning_msg}
-        return {}
```

Comment:
As this code is removed, is the documentation needs to be updated?
https://github.com/OCA/sale-workflow/blob/13.0/sale_by_packaging/readme/DESCRIPTION.rst

Reply:
Readme has been updated.